generic_config_updater/generic_updater.py

```diff
@@ -52,7 +52,17 @@ def apply(self, patch):
         self.logger.log_notice("Simulating the target full config after applying the patch.")
         target_config = self.patch_wrapper.simulate_patch(patch, old_config)
 
-        # Validate target config
+        # Validate target config does not have empty tables since they do not show up in ConfigDb
+        self.logger.log_notice("Validating target config does not have empty tables, " \
+                               "since they do not show up in ConfigDb.")
+        has_empty_tables, empty_tables = self.config_wrapper.has_empty_tables(target_config)
```

Comment:
>empty_tables [](http://example.com/codeflow?start=26&length=12)

Not need to return bool, you can check `len(empty_tables)`.

Reply:
updated